.circleci/config.yml

```diff
@@ -49,19 +80,104 @@ jobs:
     steps:
       - attach_workspace: *attach_options
       - run: npm run build:prod
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./raiden_webui/ui
+
+  build-package:
+    executor: python-executor
+    steps:
+      - attach_workspace: *attach_options
+      - run:
+          name: "Create the packages"
+          command: |
+            python setup.py sdist
+            python setup.py bdist_wheel
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./dist
+
+  deploy-github-release:
+    working_directory: ~/
+    docker:
+      - image: cibuilds/github:0.10
+    steps:
+      - attach_workspace: *attach_options
+      - when:
```

Comment:
Since you filter this job on workflow level already (line 174) this job only gets executed when there is a tag matching the pattern. So your condition here will always be true. you can just run it as a command then.